tests/chocolatey-tests/commands/choco-install.Tests.ps1

```diff
@@ -436,6 +436,52 @@ Describe "choco install" -Tag Chocolatey, InstallCommand {
         }
     }
 
+    Context "Force Installing a Package that is already installed (with a delete locked file)" {
+        BeforeAll {
+            Restore-ChocolateyInstallSnapshot
+
+            $PackageUnderTest = "installpackage"
+
+            Invoke-Choco install $PackageUnderTest --confirm
+
+            $LockedFile = [System.IO.File]::Open(
+                "$env:ChocolateyInstall\lib\$PackageUnderTest\tools\chocolateyInstall.ps1",
+                "OpenOrCreate",
+                "ReadWrite",
+                "Delete"
+            )
+
+            $Output = Invoke-Choco install installpackage --force --confirm
+        }
+
+        AfterAll {
+            $LockedFile.Close()
+        }
+
+        It "Exits with Success (0)" {
+            $Output.ExitCode | Should -Be 0 -Because $Output.String
+        }
+
+        It "Has successfully retained an install of the original package" {
+            "$env:ChocolateyInstall\lib\$PackageUnderTest\"
+        }
+
+        It "Has successfully retained the original version" {
+            "$env:ChocolateyInstall\lib\$PackageUnderTest\$PackageUnderTest.nuspec" | Should -Exist
+            [xml]$XML = Get-Content "$env:ChocolateyInstall\lib\$PackageUnderTest\$PackageUnderTest.nuspec"
+            $XML.package.metadata.version | Should -Be "1.0.0"
+        }
+
+        # This behaviour was fixed in 0.10.16

```

Comment:
We didn't release a version 0.10.16.

Reply:
THat is fair, this is mainly a copy paste from the E2E test below this one. But I can get that mention removed.